pandas/tools/pivot.py

```diff
@@ -24,12 +24,14 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
     ----------
     data : DataFrame
     values : column to aggregate, optional
-    index : a column, Grouper, array which has the same length as data, or list of them.
-        Keys to group by on the pivot table index.
-        If an array is passed, it is being used as the same manner as column values.
-    columns : a column, Grouper, array which has the same length as data, or list of them.
-        Keys to group by on the pivot table column.
-        If an array is passed, it is being used as the same manner as column values.
+    index : a column, Grouper, array which has the same length as data, or list
+        of them.
```

Comment:
In this case, maybe just shorten it a bit? (maybe 'array which has the same length' -> 'array of same length' is enough to keep within line limit?)

But if this happens more, we should maybe look if we can solve this in the numpydoc parser